lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -78,20 +82,28 @@ export class CardViewTextItemComponent implements OnChanges {
     }
 
     reset(): void {
-        this.editedValue = this.property.value;
+        this.editedValue = this.property.multiline ? this.property.displayValue : this.property.value;
         this.setEditMode(false);
     }
 
     update(): void {
         if (this.property.isValid(this.editedValue)) {
-            this.cardViewUpdateService.update(this.property, this.editedValue);
-            this.property.value = this.editedValue;
+            const updatedValue = this.prepareValueForUpload(this.property, this.editedValue);
+            this.cardViewUpdateService.update(this.property, updatedValue);
+            this.property.value = updatedValue;
             this.setEditMode(false);
         } else {
             this.errorMessages = this.property.getValidationErrors(this.editedValue);
         }
     }
 
+    prepareValueForUpload(property: CardViewTextItemModel, value: any) {
```

Comment:
missing return type

Reply:
you are actually modifying the input value maybe will be better to do it in a separate value.

Reply:
is value any?